app/basicComponents/Input.tsx

```diff
@@ -120,6 +120,8 @@ type Props = {
   maxLength?: any;
   autofocus?: boolean;
   iconLeft?: string;
+  min?: string | number;
+  max?: string | number;
```

Comment:
Why not just
```suggestion
  min?: number;
  max?: number;
```
?

Reply:
it was a reference type to the original input, changed.